queue/python/Queue.py

```diff
@@ -0,0 +1,51 @@
+__author__ = "Osamudiamen Azamegbe"
```

Comment:
It's confusing adding this here because it's not obvious from the name of the PR. The PR's name is: **Implemented addLast method in constant time**

You want to do this in a different PR.

Also,  the `Queue` and `DoublyLinkedList` classes shouldn't be in the same file. Can you use the one that has already been implemented by importing? If no, does python have any in-built data structure that mirrors what a doubly linkedlist should do.

Finally, pls read:  https://github.com/oyekanmiayo/data-structures-all-langs/blob/main/CONTRIBUTING.md. You need to add driver code for your implementations like this:  https://github.com/oyekanmiayo/data-structures-all-langs/blob/main/linkedlist/doubly/java/DoublyLinkedList.java#L33

I will update the current Queue implementation for Java to reflect this too.

Reply:
My goal wasn't to try to re-implement the doubly linked list (that is why I put a reference to its implementation in the file). 
There is no in-built Doubly Linked List in python, I can try importing the already implemented DLL, but I'll need to add an __init__.py file to the "python" folder under the doubly linked list directory.
If that is fine I can go ahead to add the __init__.py file and import the DLL. @oyekanmiayo